### DISCUSSION TITLE
You’ve got mail! 📮 [Pull Requests Mini Check-in]

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/community#53414] [Pull Requests]
Delivered from me to you - this month’s Pull Request shoutouts, releases, and community news! 

⚡**Pull Requests Praise**: Thanks to these fine folks for keeping our Community awesome ✔️
* @daneah, @arthurljones, & @mminot-yseop for providing feedback and experiences on our [discussion](https://github.com/orgs/community/discussions/49954#discussioncomment-5296774) during the `Comment on Files in Pull Request Public Beta`!  
* @joneslee0918: Jumped in on a [Code Review question](https://github.com/orgs/community/discussions/51752) with a helpful link to our docs that solved it - nice job ✨. 

ℹ️ **What’s new**: The latest Pull Requests updates
* **Commenting on files in a pull request is now generally available**: Thanks to all of your [feedback](https://github.com/orgs/community/discussions/49954) during our public beta - we’re happy to share this feature is now [generally available](https://github.blog/changelog/2023-04-11-commenting-on-files-in-a-pull-request-is-now-generally-available/) 🎉.  
* **Automatic rebases on Dependabot pull requests stop after 30 days of inactivity**: We've heard your feedback about Dependabot noisiness and are making Dependabot quieter and more focused on repositories you care about - [view the details]https://github.blog/changelog/2023-04-13-automatic-rebases-on-dependabot-pull-requests-stop-after-30-days-of-inactivity/().

🆒 **Cool stuff™️** happening around GitHub and the Community
* 🆕 **Community Interest Categories** 🧠: Take your skills that much further without even leaving the community, we added the [Programming Help](https://github.com/orgs/community/discussions/categories/programming-help?discussions_q=is%3Aopen+sort%3Adate_created+category%3A%22Programming+Help%22) and [Machine Learning](https://github.com/orgs/community/discussions/categories/machine-learning-and-mlops?discussions_q=is%3Aopen+sort%3Adate_created+category%3A%22Machine+Learning+and+MLOps%22)categories to discuss projects, programming, and more! 

Signed, sealed, and delivered 💌 - see you in May!

**Comment by JohnnyDa5** [comment]
> Delivered from me to you - this month’s Pull Request shoutouts, releases, and community news!
> 
> ⚡**Pull Requests Praise**: Thanks to these fine folks for keeping our Community awesome ✔️
> 
> * @daneah, @arthurljones, & @mminot-yseop for providing feedback and experiences on our [discussion](https://github.com/orgs/community/discussions/49954#discussioncomment-5296774) during the `Comment on Files in Pull Request Public Beta`!
> * @joneslee0918: Jumped in on a [Code Review question](https://github.com/orgs/community/discussions/51752) with a helpful link to our docs that solved it - excellent lent
![PCH WINNER Mr  Johnny E  Wellons, II](https://user-images.githubusercontent.com/116997878/236714172-7987c629-8545-4675-bbfa-d059270f602c.jpg)
✨.
> 
> ℹ️ **What’s new**: The latest Pull Requests updates
> 
> * **Commenting on files in a pull request is now generally available**: Thanks to all of your [feedback](https://github.com/orgs/community/discussions/49954) during our public beta - we’re happy to share this feature is now [generally available](https://github.blog/changelog/2023-04-11-commenting-on-files-in-a-pull-request-is-now-generally-available/) 🎉.
> * **Automatic rebates on Dependabot pull requests stop after 30 days of inactivity**: We've heard your feedback about Dependabot noisiness and are making Dependabot quieter and more focused on repositories you care about - [view the details]https://github.blog/changelog/2023-04-13-automatic-rebases-on-dependabot-pull-requests-stop-after-30-days-of-inactivity/().
> 
> 🆒 **Cool stuff™️** happening around GitHub and the Community
> 
> * 🆕 **Community Interest Categories** 🧠: Take your skills that much further without even leaving the community, we added the [Programming Help](https://github.com/orgs/community/discussions/categories/programming-help?discussions_q=is%3Aopen+sort%3Adate_created+category%3A%22Programming+Help%22) and [Machine Learning](https://github.com/orgs/community/discussions/categories/machine-learning-and-mlops?discussions_q=is%3Aopen+sort%3Adate_created+category%3A%22Machine+Learning+and+MLOps%22)categories to discuss projects, programming, and more!
> 
> Signed, sealed, and delivered 💌 - see you in May!

**Comment by JohnnyDa5** [comment]
https://github.com/community/community.git

**Reply by JohnnyDa5** [reply]
![image](https://user-images.githubusercontent.com/116997878/236714247-a2c2798e-3a77-4b44-8e81-7b331334fb46.png)
WhoSaiDaThere@gmail.com✔🙌💵💵💵💵💵💵💵💵💵💵💵💵💵💵💵💵

**Reply by JohnnyDa5** [reply]
Mr. Johnny Edward Wellons, II 🏆 GRANDSUPERPRIZEWINNER
II

**Comment by queenofcorgis** [comment]
The phone is ringing 📞  - will you pick up for May's check-in? [View the newest check-in post via this link](https://github.com/orgs/community/discussions/55875).
